lib/child_process.js

```diff
@@ -352,6 +352,8 @@ function normalizeSpawnArguments(file, args, options) {
 
   if (Array.isArray(args)) {
     args = args.slice(0);
+    for (var i = 0; i < args.length; i++)
+      args[i] = `${args[i]}`;
```

Comment:
That calls `args[i][Symbol.toPrimitive]` (if there is one) with `'default'` instead of `'string'`, thus breaking compatibility.